### ISSUE TITLE
AttributeError: 'Base' object has no attribute 'base_id'

### ISSUE THREAD

**Issue by robfitzgerald** [NREL/hive#128]
while running hive and producing an error due to a missing station/base relationship, the error message itself raised an exception. this is because the message attempts to print `{base.base_id}` when base only has the field `{base.id}`.

### stack trace

```
Traceback (most recent call last):
  ...
  File "/Users/rfitzger/anaconda3/envs/hive-evse/lib/python3.7/site-packages/nrel/hive/app/hive_cosim.py", line 111, in crank
    next_state, unmerged_events = ft.reduce(run_step, steps, initial)
  File "/Users/rfitzger/anaconda3/envs/hive-evse/lib/python3.7/site-packages/nrel/hive/app/hive_cosim.py", line 90, in run_step
    rp1 = rp0.u.apply_update(rp0)
  File "/Users/rfitzger/anaconda3/envs/hive-evse/lib/python3.7/site-packages/nrel/hive/state/simulation_state/update/update.py", line 96, in apply_update
    pre_step_result.runner_payload.s, pre_step_result.runner_payload.e
  File "/Users/rfitzger/anaconda3/envs/hive-evse/lib/python3.7/site-packages/nrel/hive/state/simulation_state/update/step_simulation.py", line 98, in update
    self.ordered_instruction_generators, sim_with_drivers_updated, env
  File "/Users/rfitzger/anaconda3/envs/hive-evse/lib/python3.7/site-packages/nrel/hive/dispatcher/instruction_generator/instruction_generator_ops.py", line 132, in generate_instructions
    driver_result = result.add_driver_instructions(simulation_state, environment)
  File "/Users/rfitzger/anaconda3/envs/hive-evse/lib/python3.7/site-packages/nrel/hive/dispatcher/instruction_generator/instruction_generator_ops.py", line 93, in add_driver_instructions
    (),
  File "/Users/rfitzger/anaconda3/envs/hive-evse/lib/python3.7/site-packages/nrel/hive/dispatcher/instruction_generator/instruction_generator_ops.py", line 88, in <lambda>
    self.instruction_stack.get(v.id),
  File "/Users/rfitzger/anaconda3/envs/hive-evse/lib/python3.7/site-packages/nrel/hive/state/driver_state/autonomous_driver_state/autonomous_available.py", line 66, in generate_instruction
    return av_charge_base_instruction(my_vehicle, sim, env)
  File "/Users/rfitzger/anaconda3/envs/hive-evse/lib/python3.7/site-packages/nrel/hive/state/driver_state/driver_instruction_ops.py", line 218, in av_charge_base_instruction
    log.error(f"could not find station {my_base.station_id} for base {my_base.base_id}")
AttributeError: 'Base' object has no attribute 'base_id'
```

**Comment by nreinicke** [comment]
This was addressed in 84f6a6e but hasn't made it into a public release. There's been about 50 commits to main since our latest release so I think we could justify a minor release of v1.1.

**Comment by nreinicke** [comment]
This is fixed in v1.1.0
